WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -39,6 +39,17 @@ public object[] GetUnspentCoinList()
 			}).ToArray();
 		}
 
+		[JsonRpcMethod("createwallet")]
+		public object CreateWallet(string walletName, string password)
+		{
+			var walletGenerator = new WalletGenerator(Global.WalletsDir, Global.Network);
+			walletGenerator.TermsAccepted = true;   // Global.WalletService is {};
```

Comment:
Shouldn't we remove the comment.

Reply:
It is not in the added code, it is just to demonstrate that the rpc interface displays exactly the same error messages that the UI.